app/models/mission.rb

```diff
@@ -34,7 +34,7 @@ class Mission < ApplicationRecord
 
   scope(:sorted_by_name, -> { order("name") })
   scope(:sorted_recent_first, -> { order("missions.created_at DESC") })
-  scope(:for_user, ->(u) { where("missions.id IN 
+  scope(:for_user, ->(u) { where("missions.id IN
```

Comment:
Is there a way to write this query in ActiveRecord?

Something like `where(id: Assignment.where(user_id: u.id).pluck(:mission_id))` (this should pick up paranoia's default queries)?

I think we should be working to avoid hand-crafted SQL when possible since it helps us avoid a lot of the types of migration pains we experienced switching to  postgres and then to UUIDs.

Reply:
Sure I can try it. Or even `where(id: user.assignments.pluck(:id))`, right? 

This was just a whitespace change but I guess that's my rotten luck :P

Reply:
Actually turns out this method is a crappier version of user.missions and is only used once so I got rid of it :D